src/coldcard.ts

```diff
@@ -176,11 +176,8 @@ class ColdcardMultisigSettingsFileParser extends ColdcardInteraction {
    * add a field for rootFingerprint (it can sometimes be calculated
    * if not explicitly included)
    *
-   * @param {Object} file JSON file exported from Coldcard
-   * @returns {Object} the parsed response
-   *
    */
-  parse(file) {
+  parse(file: any) {
```

Comment:
Based on the code below I think this works as a type declaration for this arg.

```suggestion
  parse(file: Record<string, unknown> | string) {
```

Reply:
Changed in 35668a8805582ecc715ddc5e5214ac4de91c22ff